src/ADNLPProblems/toint.jl

```diff
@@ -0,0 +1,40 @@
+export toint
+
+function toint(; n::Int = default_nvar, type::Type{T} = Float64, kwargs...) where {T}
```

Comment:
There seems to be an inconsistency between the AD version and the JuMP version:
```
julia> ju_nlp1 = MathOptNLPModel(OptimizationProblems.PureJuMP.toint());

julia> ad_nlp1 = OptimizationProblems.ADNLPProblems.toint();

julia> obj(ad_nlp1, ad_nlp1.meta.x0) == obj(ju_nlp1, ju_nlp1.meta.x0)
false
```
The other two seem correct.